index.html

```diff
@@ -0,0 +1,154 @@
+					<a href="mailto:mrrobinhodge@gmail.com"><img src="email.png" alt="An email thumbnail"></a>
+					<a href="https://www.linkedin.com/in/robin-hodge/" target="_blank"><img src="linkedin.png" alt="A linkedin thumbnail"></a>
+					<a href="https://github.com/horsenelly" target="_blank"><img src="github.png" alt="A github thumbnail"></a>
+				</div>
+			</div>
+		</div>
+		</div>
+		<nav id="nav">
+			<div class="nav-left-container">
+				<a href="#about">About</a>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#contact">Contact</a>
+				<a href="https://docs.google.com/document/d/1G8ed6eSbtD6e5KjoC9QAf5T7TktOjEkubLuD5ih2ifs/edit" target="_blank">My CV</a>
+		</nav>
+		<div class="intro">
+			<img src="https://images.mubicdn.net/images/cast_member/2552/cache-207-1524922850/image-w856.jpg" alt="An image of me"> 
+			<div class="intro-texts">
+				<h2>Hi!<h2>
+				<h1>Welcome to <br>Robin's portfolio</h1>
+				<h3>I am an aspiring full stack developer, and recent graduate from iO Academy</h3>
+			</div>
+		</div>
+		<div class="about" id="about">
+			<div class="about-me">
+				<div class="about-me-text">
+					<h2>About</h2>
+					<p>Hi, I'm Robin. I have spent almost a decade teaching maths and economics, and have recently upskilled myself to move into software development. I have just finished studying on the <a href="https://io-academy.uk/courses/full-stack-track/" target="_blank" class="full-stack-track-link">iOAcademy Full Stack Track</a> bootcamp. I am a hard working and friendly team player, with excellent interpersonal skills, and I am a particalarly strong communicator. I have been very successful in my time as a teacher, and I look forward to applying my many transferrable skills from the education sector into the world of coding.
+					</p>
+				</div>
+				<img src="https://i2-prod.mirror.co.uk/incoming/article22666630.ece/ALTERNATES/s615b/1_r-than-ever-in-sexy-new-campaign-for-brioni-nbsp-pics-793090.jpg" alt="A portrait photo of me on black and white">
+			</div>
+		</div>
+		<div class="portfolio" id="portfolio">
+			<div class ="portfolio-title-container">
+				<h2>Portfolio</h2>
+				<h3>Here is a small selection of some projects I have worked on</h3>
+			</div>
+			<div class="projects-container">
+				<div class = "project">
+					<h3>Project Title</h3>
+					<div class="project-content">
+						<img src="https://media.sketchfab.com/models/3efb1bf4e0ce4c97b6538b48d0ba2ecc/thumbnails/22df3424b68845e29da5f48fc28bb1f7/8d639bf37d74479b9f74c9d3958c5c7f.jpeg" alt="A nice image for the first project">
+						<div class="project-words">
+							<p>This is the first thing I would like to say about the project. This is what it was, what it did, and also this is how it did it. I like this particular thing about it, for this reason that I am saying now. It was interesting because of these reasons mostly.
+							</p>
+							<div class="project-links-container">
+								<a href="github-link.org">Github</a>
```

Comment:
it is better for the non working links to use `href="#"`, rather than a made up link, in case your made up link turns out to be an existing dodgy website

Reply:
same for the other one

Reply:
done!